branch.go

```diff
@@ -182,7 +253,7 @@ func createBranchesWithNames(db *gorm.DB, projectID, tokenID uint, branchNames [
 func deleteBranchesByNames(db *gorm.DB, projectID uint, branchNames []string) error {
 	return db.
 		Where(&database.Branch{ProjectID: projectID}, database.BranchFields.ProjectID).
-		Where(database.BranchColumns.Name+" IN ?", branchNames).
+		Where(database.BranchColumns.Name+" IN ?", stringSliceToInterfaces(branchNames)).
 		Delete(&database.Branch{}).
```

Comment:
This one was an annoying issue.

In the documentation, it states that you can pass in a slice array: https://gorm.io/docs/query.html#String-Conditions

But with some testing that resulted in this:

```sql
DELETE FROM `branch`
WHERE `branch`.`branch_id` IN (
  "name IN ?",
  ("main","feature/test","feature/test2","feature/fixed-thing","dev")
) AND `branch`.`project_id` = 1
```

Converting the type from `[]string` to `[]interface{}` makes it work.
Will write a note about this in the iver-wharf.github.io docs down in the "Things to keep in mind" section

Reply:
I've opened a PR on it here:

https://github.com/iver-wharf/iver-wharf.github.io/pull/79